src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -835,6 +883,7 @@ private MeasureInfo CreateGenericMeasureInfo(Statistics stat, bool shouldUseGene
                 gmi.Count = stat.count;
                 gmi.Sum = sum;
                 gmi.Average = average;
+                gmi.StdDeviation = stdDeviation;
```

Comment:
Is there a benefit if we put `stdDeviation` out too?

Reply:
I'm not sure what you mean, can you elaborate?

Reply:
Oh, I see what you mean; standard deviation doesn't seem to be as necessary. I don't think people would usually need it.

Reply:
Currently, just deleting that line messes up the code. I'll try to understand it better and try again to work on this issue.